clouddriver-kubernetes/src/integration/java/com/netflix/spinnaker/clouddriver/kubernetes/it/InfrastructureIT.java

```diff
@@ -829,4 +831,345 @@ public void shouldGetRawResources() throws InterruptedException, IOException {
             + appName
             + "/rawResources to return valid data");
   }
+
+  @DisplayName(
+      ".\n===\n"
+          + "Given two kubernetes network policies\n"
+          + "When sending get securityGroups\n"
+          + "Then response should contain a list with security groups of size 2\n===")
+  @Test
+  public void shouldListSecurityGroups() throws InterruptedException, IOException {
+    // ------------------------- given --------------------------
+
+    String appName = "getmanifest";
+    System.out.println("> Using namespace " + ns);
+
+    KubeTestUtils.deployIfMissing(
+        baseUrl(), ACCOUNT1_NAME, ns, "networkPolicy", NETWORK_POLICY_1_NAME, appName, kubeCluster);
+
+    KubeTestUtils.deployIfMissing(
+        baseUrl(), ACCOUNT1_NAME, ns, "networkPolicy", NETWORK_POLICY_2_NAME, appName, kubeCluster);
+
+    KubeTestUtils.repeatUntilTrue(
+        () -> {
+          // ------------------------- when --------------------------
+          System.out.println("> Sending get securityGroups request");
+          Response resp = get(baseUrl() + "/securityGroups");
+
+          // ------------------------- then --------------------------
+          System.out.println(resp.asString());
+
+          resp.then().statusCode(200);
+          List securityGroupList = resp.jsonPath().getList(ACCOUNT1_NAME + ".kubernetes." + ns);
```

Comment:
Using raw types shows compilation warning like these (there are many):

```
clouddriver-kubernetes/src/integration/java/com/netflix/spinnaker/clouddriver/kubernetes/it/InfrastructureIT.java:863: warning: [rawtypes] found raw type: List
          List securityGroupList = resp.jsonPath().getList(ACCOUNT1_NAME + ".kubernetes." + ns);
```

Let's use `List<Object>` instead of raw types